src/clr-core/button/base-button.element.scss

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
```

Comment:
Base button SCSS was split out so that the icon button wouldn't be impacted by CSS intended for the full-featured button.